package.json

```diff
@@ -52,6 +53,11 @@
   "resolutions": {
     "csstype": "^2.6.18"
   },
+  "overrides": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "prettier": "^1.19.1"
```

Comment:
This isn't necessary if we update our `prettier` dependency, but newer versions want to reformat the entire codebase so I didn't want to do this now